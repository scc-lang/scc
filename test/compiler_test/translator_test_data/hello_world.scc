std::println("hello world!");
std::println("hello\nwor\3ld!");