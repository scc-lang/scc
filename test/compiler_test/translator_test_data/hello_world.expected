// scc autogenerated file.

import scc.std;

int main()
{
    scc::std::println("hello world!");
    scc::std::println("hello\012wor\003ld!");
    return 0;
}
